app/models/event.rb

```diff
@@ -6,6 +6,13 @@ class Event < ActiveRecord::Base
   validates :start_on, presence: true
   validates :end_on, presence: true
   validates :name, presence: true
+  validate :validate_end_on_before_start_on
+
+  def validate_end_on_before_start_on
```

Comment:
Let's name this something simpler, like maybe `#start_before_end`, then our DSL will read `validate :start_before_end` WDYT?